README.md

```diff
@@ -3156,11 +3156,20 @@ be checked for.
 An array of allowable license values or `true` to allow any license text.
 If present as an array, will be used in place of SPDX identifiers.
 
+<a name="eslint-plugin-jsdoc-rules-check-values-options-5-licensepattern"></a>
+##### <code>licensePattern</code>
```

Comment:
Can the name for this option be more clear that it deals with the license description instead of license name?

Reply:
It actually does deal with the license name though. It finds the license name out of the whole tag description.